tests/phpunit/elementor/core/experiments/test-manager.php

```diff
@@ -193,9 +209,90 @@ public function test_remove_feature() {
 		$this->assertNull( $feature );
 	}
 
-	private function add_test_feature( array $args = [] ) {
-		$experiments = $this->elementor()->experiments;
+	public function test_validate_dependency__ensure_error_dependency_not_available() {
+		// Arrange.
+		$test_feature_data = [
+			'name' => Module_A::instance()->get_name(),
+			'state' => Experiments_Manager::STATE_INACTIVE,
+			'dependencies' => [
+				Module_B::class,
+			],
+		];
+
+		$this->add_test_feature( $test_feature_data );
+
+		// Assert.
+		$this->expectException( \WPDieException::class );
```

Comment:
Why not `Dependency_Exception`? Here and below.

Reply:
It's not working WP test env make it `WPDieException`.